apps/feedahelp-mobile/components/login/Form.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { View, TextInput, StyleSheet, Text, TouchableOpacity } from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+
+export default function Form() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  return (
+    <View style={styles.formContainer}>
+      <View style={styles.inputWrapper}>
+        <Ionicons name="mail-outline" size={20} style={styles.icon} />
+        <TextInput
+          style={styles.input}
+          placeholder="Email"
+          placeholderTextColor="#ef8d77"
+          keyboardType="email-address"
+          autoCapitalize="none"
+          value={email}
+          onChangeText={setEmail}
+        />
+      </View>
+      <View style={styles.inputWrapper}>
+        <Ionicons name="lock-closed-outline" size={20} style={styles.icon} />
+        <TextInput
+          style={styles.input}
+          placeholder="Password"
+          placeholderTextColor="#ef8d77"
+          secureTextEntry={true}
+          value={password}
+          onChangeText={setPassword}
+        />
+      </View>
+
+      <TouchableOpacity style={styles.button}>
+        <Text style={styles.buttonText}>Log in and remember Password</Text>
+      </TouchableOpacity>
+
+      <TouchableOpacity>
+        <Text style={styles.linkText}>Log in without remembering Password</Text>
+      </TouchableOpacity>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
```

Comment:
get this out of here:
Add another `form.style.tsx`

and `export` it from there

Reply:
I will have to do 2nd review once you move this out side